src/providers/wcs/qgswcscapabilities.h

```diff
@@ -114,6 +123,9 @@ class QgsWcsCapabilities : public QObject
      */
     void coverageParents( QMap<int, int> &parents, QMap<int, QStringList> &parentNames ) const;
 
+    // Parse metadata element from the document
+    void parseMetadataLink( QDomElement const &e, QgsWcsMetadataLinkProperty &metadataLink );
```

Comment:
QGIS coding convention is:
```suggestion
    void parseMetadataLink( const QDomElement &element, QgsWcsMetadataLinkProperty &metadataLink );
```

i.e. const comes first, and all abbreviations in attribute names must be avoided

Reply:
The thing is, this whole class already follows non-standard practice.

 @Samweli would you mind switching this all to the proper style?

Reply:
@m-kuhn alright, will be working on it.